app/views/shared/_paginate_and_filter_by_facility.html.erb

```diff
@@ -23,5 +23,8 @@
       %>
     </div>
   </div>
+  <% (@search_filters || []).each do |value| %>
+    <%= form.hidden_field "search_filters[]", value: value %>
```

Comment:
Here is the other side of the state madness: we need to include search fields as hidden fields on the facilities  / pagination filter. Two forms using GET requests and managing state is nasty. 😢

Reply:
Oh yes, the `form=` will fix this, for sure :)